ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/views/HiveMaterializedViewUtils.java

```diff
@@ -277,6 +276,20 @@ public void visit(RelNode node, int ordinal, RelNode parent) {
             materialization.getAst());
   }
 
+  private static HiveRelOptMaterialization augmentMaterializationWithTimeInformation(
```

Comment:
nit: pls add a brief comment for this important method.

Reply:
This method is part of the
```
public static HiveRelOptMaterialization augmentMaterializationWithTimeInformation(
      HiveRelOptMaterialization materialization, String validTxnsList,
      MaterializationSnapshot snapshot) throws LockException
```
method.
These utility methods construct a rule parameterize it with the saved Materialized view creation metadata and apply it to the MV query plan.
I refactored these to extract common parts and make it more readable.